exec_test.go

```diff
@@ -8,6 +8,7 @@ import (
 	"bufio"
 	"compress/bzip2"
 	"fmt"
+	"internal/testenv"
```

Comment:
This won't compile.

Reply:
I've cherry picked your commit c28f921075860bcf095babc30a357bbc02c4068f from main into this PR.